corehq/apps/style/static/style/less/bootstrap3/forms.less

```diff
@@ -94,6 +94,11 @@ legend .subtext {
   }
 }
 
+// Static width for select2 widgets, which otherwise grow too large on case management pages
+.form-control.select2-container {
```

Comment:
was gonna comment the same thing biyeun mentioned. worth checking what the selects look like on reports, though i'd imagine that limiting it to 210 isn't that big of a deal anyways